Content.Client/Nyanotrasen/Mail/MailSystem.cs

```diff
@@ -30,17 +29,21 @@ namespace Content.Client.Mail
     /// </remarks>
     public sealed class MailJobVisualizerSystem : VisualizerSystem<MailComponent>
     {
-        [Dependency] private readonly IPrototypeManager _prototype = default!;
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+        [Dependency] private readonly SpriteSystem _stateManager = default!;
+        [Dependency] private readonly SpriteSystem _spriteSystem = default!;
```

Comment:
why is this declared twice? you only need 1

Reply:
Delta merge, will fix